entries.py

```diff
@@ -234,6 +274,11 @@
 	pronunciation="", 
 	seealso="")
 
+WES = GlossEntry("Workflow Execution Service",
+	furtherreading="",
+	definition="A standardized API developed by [GA4GH] for describing a standard programmatic way to run and manage workflows.",
```

Comment:
What are some examples of platforms that can utilize this API?

Reply:
The new Dockstore CLI integration with AWS Genomics utilizes WES.

Reply:
Actually, here's a better question: In 1.12 Dockstore we worked with Amazon to launch workflows in AWS via WES via the Dockstore API. Do you know if other platforms such as Terra and SB use WES too?

Reply:
I don't think so, but we should confirm on the Dockstore slack? I believe you can share run logs, etc using WES and I don't believe we get that with either of those launch-with partners at the moment.

Reply:
It looks like the Dockstore CLI and maybe Cromwell are the only current public examples, so I think this definition is sufficient.